extensions/wikia/ContentWarning/ContentWarningController.class.php

```diff
@@ -10,7 +10,7 @@ class ContentWarningController extends WikiaController {
 	 * render index template
 	 */
 	public function index() {
-		$domain = str_replace('http://', '', $this->wg->Server );
+		$domain = preg_replace( '!^https?://!', '', $this->wg->Server );
 		$this->body = wfMsgExt( 'content-warning-body', array('parse'), $domain );

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[project-id]: wikia:app